recipes/enpt_enmapboxapp/meta.yaml

```
@@ -0,0 +1,41 @@
+{% set name = "enpt_enmapboxapp" %}
+{% set version = "0.7.6" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/enpt_enmapboxapp-{{ version }}.tar.gz
+  sha256: ada1c75c6c9a562bde374542aff4774b45e22e6f62f9f5757966cb2905e9cb89
+
+build:
+  skip: true  # [py<37]
```

I think this is pure Python and could be noarch?
```suggestion
  noarch: python
```

---

I there anything I need to do to make the CI run for noarch (Linux only)? It still runs for all platforms.

---

Well it was doing that before too. What keeps it from working on other platforms? If it's missing dependencies, we'll just ignore the failures.

---

It should be platform-independent. I meant in case of a noarch build, the build is usually running on Linux only. But it seems the CI is still trying to build it on all platforms separately. Probably, there is a certain command to re-render the meta.yaml so that the CI respects the `noarch`?

---

CI here will always run all 3. When the feedstock gets created, it will only have the Linux build.